packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-template #step1>
+    <nui-textbox
+        i18n-caption
+        caption="First"
+        name="hint"
```

Comment:
Maybe this instead to be consistent with the checkbox name below:

```suggestion
        name="nui-demo-textbox"
```